src/VectorizeLoops.cpp

```diff
@@ -118,11 +119,30 @@ Interval bounds_of_lanes(const Expr &e) {
         } else if (is_negative_const(r->stride)) {
             return {r->base + last_lane_idx * r->stride, r->base};
         }
+    } else if (const LE *le = e.as<LE>()) {
+        // The least true this can be is if we maximize the LHS and minimize the RHS
```

Comment:
super-duper-nit: each of these comment lines is a sentence, and should end with a `.`